addons/crm/static/src/js/fields/many2one_avatar_leader_user.js

```diff
@@ -0,0 +1,103 @@
+import { Many2XAvatarUserAutocomplete } from "@mail/views/web/fields/avatar_autocomplete/avatar_many2x_autocomplete";
+import {
+    Many2OneAvatarUser,
+    many2OneAvatarUserField,
+    Many2OneAvatarUserField,
+} from "@mail/views/web/fields/many2one_avatar_user_field/many2one_avatar_user_field";
+import { onWillUpdateProps, useState } from "@odoo/owl";
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+class Many2XAvatarUserLeaderAutocomplete extends Many2XAvatarUserAutocomplete {
+    static props = {
+        ...Many2XAvatarUserAutocomplete.props,
+        leaderState: Object,
+    };
+
+    /**
+     * @override
+     */
+    buildRecordSuggestion(request, record) {
```

Comment:
Why this code ? I thought it was natively supported ... do you really have to do something for Many2X... fields ? I see it is supported somehow in relational_utils.js .